roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -0,0 +1,219 @@
+---
+- name: Verify that kubeconfig is provided
+  ansible.builtin.assert:
+    that:
+      - kbpc_kubeconfig is defined
+      - kbpc_kubeconfig|length
+    fail_msg: "kubeconfig is required to run certsuite"
+
+- name: Fail if certsuite version is lower than v5.1.3
+  when:
+    - kbpc_version is version("v5.1.3", "<")
+    - kbpc_version != "HEAD"
+  ansible.builtin.fail:
+    msg: "k8s_best_practices_certsuite role is no longer compatible with versions lower than v5.1.3"
```

Comment:
if this role is new then "no longer compatible" makes no sense and also it doesn't *what* isn't compatible with, change to something like "This role is not compatible with X version < v.5.1.3"

Reply:
good catch, this is because I just copied the same task from cnf_cert role, I'll change that